tracing.go

```diff
@@ -252,10 +253,26 @@ func ExtractInboundSpan(ctx context.Context, call *InboundCall, headers map[stri
 
 func setPeerHostPort(span opentracing.Span, hostPort string) {
```

Comment:
PS: I opened an issue in OpenTracing (https://github.com/opentracing/opentracing.github.io/issues/116) to support a standard tag `peer.hostPort` so that the parsing logic could be consolidated in the tracer, instead of being duplicated across the RPC libraries instrumented with OT.